cypress/support/step-definitions/grid-steps.js

```diff
@@ -38,6 +44,16 @@ Then('pod {int} has width from column {int} to column {int}', (index, colStart,
     .and('have.css', 'grid-column-end', `${colEnd}`);
 });
 
+Then('justify pod {int} has style for justify {word}', (index, justify) => {
+  justifyGridComponent().children('div').children('div').eq(index)
```

Comment:
```suggestion
  justifyGridComponent().children('div').children('div').eq(positionOfElement(position))
```
position should be a string like `first`, `second` etc.

Reply:
fixed